Gemfile

```diff
@@ -88,6 +88,9 @@ group :development, :test do
   gem 'simplecov', require: false
   gem 'yard', '~> 0.9.14'
   gem 'yard-activesupport-concern'
+
+  # Ruby static code analyzer and code formatter
+  gem "rubocop", require: false
```

Comment:
should always specify a major version, e.g.: `gem xx, "~> 1.2.0" `, you can copy latest version requirement from rubygems.org